src/lib/pages/backoffice/Acquisition/Order/OrderSearch/OrderList/OrderListEntry.js

```diff
@@ -159,18 +160,20 @@ export default class OrderListEntry extends Component {
             to={AcquisitionRoutes.orderDetailsFor(record.metadata.pid)}
             data-test={`navigate-${record.metadata.pid}`}
           >
-            <AcquisitionOrderIcon />
-            Order: {docTitle ? docTitle : record.metadata.pid}
+            <Truncate lines={2} width="500px">
```

Comment:
Comment: Ideally we try to use relative units, however since the backoffice is not meant to be fully responsive, I think it's good enough.

Reply:
could you check if relative units work? fe 80% ?

Reply:
Yes, I used relative units at first and it worked on 50% but it was not responsive enough if the width was decreased a bit too much.
(My reasoning was that back office wouldn't be used in mobile view)
I went with what we already have in the Documents page i.e. 500px